src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -311,13 +312,13 @@ class SelectorDropdown extends React.Component<Props, State> {
                 onPaste={this.handleInput}
                 ref={this.selectorDropdownRef}
             >
-                {React.cloneElement(selector, { inputProps })}
-                {isOpen && (
-                    <div className={`${OVERLAY_WRAPPER_CLASS} is-visible`}>
+                <PopperComponent placement={PLACEMENT_BOTTOM_START} isOpen={isOpen}>
+                    {React.cloneElement(selector, { inputProps })}
+                    <div className="SelectorDropdown-overlay-wrapper">
```

Comment:
descendant element would be camelcase `SelectorDropdown-overlayWrapper`

but maybe wrapper isn't necessary: `SelectorDropdown-overlay`